CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 30.5.3
```

Comment:
This needs to be moved back to `## Unreleased` (I think a rebase causes this to happen as your change stays on the same line, but the heading changes)

Reply:
Done.